### ISSUE TITLE
Auto-detect timezones in event creation

### ISSUE THREAD

**Issue by saikat** [Bernie-2016/ground-control#319]
Another user request.  I think making the timezone blank is probably good enough, but we do have moment-timezone and could load some limited timezone data into the client (or use something like https://developers.google.com/maps/documentation/timezone/intro?hl=en) to auto-populate the timezone dropdown.  I don't think this is that high a priority though, but would be a nice to have.

**Comment by jlegrone** [comment]
This would only make sense on the public event creation tool, yes?

**Comment by saikat** [comment]
I think both if it was possible to set the timezone from the event address
in the admin tool

**Comment by jlegrone** [comment]
It certainly would be. We'll want to move the location section above the date section in that case too.

**Comment by jlegrone** [comment]
The files to modify are in src/frontend/public/admin/events. Look for create_event.html and create_event_public.html (I'd like to roll these into a handlebars template at some point, but for now just dup your changes in both... or set up a handlebars template). ;)

1. Change the order of sections in the form to put 'Where' above 'When' in the form (so that users don't pick a timezone before we can select one for them)

2. Hook into the `fillInAddress()` function and use some library to determine the timezone at that address and update the value for the time zone select appropriately. We're already loading the google maps api, so we may be able to use https://developers.google.com/maps/documentation/timezone/intro
